src/cmd/tools/m3ctl/main/http/checker.go

```diff
@@ -0,0 +1,19 @@
+package http
```

Comment:
IMHO, you should not name this package `http` because reading the code gets confusing since this name already exists in the stdlib. Instead use a name that is specific to your needs.